src/test/scenarios/managed-network/configuration/readme.az.md

```diff
@@ -28,16 +28,12 @@ az:
 python-sdk-output-folder: "$(output-folder)/src/managed-network/azext_managed_network/vendored_sdks/managed-network"
 
 clicommon:
-    naming:
-        default:
-            singularize:
-              - operationGroup
-              - operation
     cli-directive:
     # directive on operationGroup
-      - select: 'operationGroup'
+      - select: 'operation'
```

Comment:
>select [](start = 8, length = 6)

FYI. this is optional, usually can be figured out automatically by where below

Reply:
I see